src/test/datascience/mockInputBox.ts

```diff
@@ -47,6 +47,14 @@ export class MockInputBox implements InputBox {
                     this.didHideEmitter.fire();
                 } else {
                     this.didAcceptEmitter.fire();
+
+                    // Accept may cause validation failure. Check for that
+                    setTimeout(() => {
+                        if (this.validationMessage) {
+                            this.value = this.validationMessage;
+                            this.didHideEmitter.fire();
+                        }
+                    }, 100);
```

Comment:
This timeout seems pretty arbitrary. Is it possible for validationMessage to be populated after 100ms? would an interval be more reliable?

Reply:
This is only used in a unit test so the only way this timeout doesn't work is if you're debugging the unit test.